### PR TITLE
Mainpageupdate

### PR DESCRIPTION
**Pull request by Pranaav003**
Updating the Main Page to be updated to current

### CODE REVIEW

**Comment by zeshengchen** [comment]
Please provide the description of this pull request.

File: frontend/originaltemplate.html
Comment (jude-lieb) [inline frontend/originaltemplate.html:1]: Is this file actually being used or can it be removed?

Comment (Pranaav003) [reply frontend/originaltemplate.html:1]: It can be removed, I was thinking to remove it after the request is pulled in.

File: frontend/mainpage.html
Comment (jude-lieb) [inline frontend/mainpage.html:1]: Perhaps the CSS could be separated into its own file.

**Review by jude-lieb — APPROVED** [review]
Ready to go

**Comment by zeshengchen** [comment]
@cgibson1421 Please peer review this pull request very soon!

**Pull request merged**